README.md

```diff
@@ -48,5 +52,13 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 ## Runtime Analysis
 
 What is the worst-case asymptotic time complexity of your implementation? What
-is the worst-case asymptotic memory complexity? Add your answer, including your
-reasoning, to this markdown file.
+is the worst-case asymptotic memory complexity? 
+
+We have a time complexity and memory complexity of $\Theta(n)$.
+Building the initial route list is a time complexity of $\Theta(n)$. Then we run an 
+exhaustive loop to check if the distance is getting shorter. Inside this loop, 
+we determine whether a swap would be more efficient and if thats the case then we swap.
+This swap costs us another time of $\Theta(n)$. We run this a minimum of 10 times but
+constants dont count. This is where we get the time complexity of $\Theta(n)$.
```

Comment:
But you stop if no improvements have been observed for 10 iterations, right? Is that really linear in the input size?